src/rule.cc

```diff
@@ -210,7 +214,8 @@ rule_json_callback (const char *locator, const char *value, void *data)
         }
         else {
             is_email = streq(self->parser.action, "EMAIL") ||
-                            streq(self->parser.action, "SMS");
+                            streq(self->parser.action, "SMS") ||
+                            streq(self->parser.action, "AUTOMATION");
```

Comment:
Why is a variable called `is_email` set if the action is `AUTOMATION`? Should it care? Should that variable be renamed?

Reply:
Good point. Why SMS is handled here too?
I was tempted to rename that var., but i wouldn't change that now. The meaning of that boolean value is 'action recognized'.